RootCommandBuilder.cs

```diff
@@ -0,0 +1,47 @@
+using System.Net.Http;
+using System.CommandLine;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.CommandLine.Invocation;
+using System.Threading;
+using System;
+
+namespace project
+{
+    public class RootCommandBuilder
+    {
+        private readonly IEnumerable<IFeature> features;
+
+        public RootCommandBuilder(IEnumerable<IFeature> features)
```

Comment:
This is the magic of the DI container - the collection will be populated with all the registered features